index.d.ts

```diff
@@ -131,4 +131,5 @@ declare namespace EventEmitter {
   export const EventEmitter: EventEmitterStatic;
 }
 
-export = EventEmitter;
```

Comment:
I think this is a breaking change for backward compatibility.

Reply:
The new code won't work with this syntax I believe. 
Exporting it as both default export and named export should sufficiently cover it.

I tested it locally and it is working.

If you want, I can add some type level tests. But for that, I'll add some dev dependencies so the test can be executed.

Reply:
Does it work if you use

```js
import EventEmitter = require('eventemitter3');
```

and

```js
import x from 'eventemitter3';
```

?

Reply:
Yes. That's the `export default` provides.

With `export =`, when using like `import { EventEmitter } from 'eventemitter3'`, it will error like:

```
error TS2749: 'EventEmitter' refers to a value, but is being used as a type here. Did you mean 'typeof EventEmitter'?
```

The `export =` syntax is basically the TypeScript wrokaround way to describe ES5 in CommonJS. It is really deprecated as `import module = require('..')` is not a valid ESM code and will not work in most cases.

https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require